content/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.md

```diff
@@ -43,9 +38,19 @@ As the name suggests, this is a collaborative method for recovering data. For ma
 
 Before installing and using the Collaborative Recovery CorDapps, you need to know:
 
-* Your Business Network disaster recovery policy
-* The Corda platform requirements
-* How and when the **LedgerSync** and **LedgerRecover** CorDapps should be used
+* Your Business Network disaster recovery policy.
+* The Corda platform requirements.
+* How and when the **LedgerSync** and **LedgerRecover** CorDapps should be used.
+
+## In this version
+
+You can now use the `skipSizeQuery` configuration in the Ledger Recover CorDapp in cases where you cannot grant `SELECT` permission by the PostGres admin to check the file size of the exported transactions before exporting them. This check normally verifies that the size of files to be exported does not exceed your node’s disk size.
```

Comment:
A minor thing but should "Ledger Recover CorDapp" be `LedgerRecover` (as per https://docs.r3.com/en/platform/corda/4.8/enterprise/node/collaborative-recovery/introduction-cr.html#ledgerrecover)?

Reply:
Also: PostGres > PostgreSQL